tests/test_picai/test_case.py

```diff
@@ -0,0 +1,63 @@
+import os
+import tempfile
+from pathlib import Path
+
+import numpy as np
+import SimpleITK as sitk
+
+from research.picai.preprocessing import PicaiCase, PreprocessingSettings
+
+
+def test_read_and_write_picai_case() -> None:
+    X, Y, Z = 32, 64, 128
```

Comment:
Very minor, but I'd maybe make these a little small to reduce overhead slightly. Maybe 8, 16, 32?